recipes/kernax/meta.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: kernax
+  version: 0.0.1
+
+source:
+  # git_url: https://gitlab.com/drti/kernax.git
+  url: https://gitlab.com/drti/kernax/-/archive/main/kernax-main.tar.gz
+  sha256: bd3896bbb15120ebc3c7732899451f924dba8367d1b1119a61018295664f8fd1
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]  
```

Comment:
Does this package not build under windows? If it's just a missing dependency, I'd say we should make this `noarch: python`:
https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Yes, it didn't build under windows because the latest versions of jax (and jaxlib) are not available yet under that platform, if I understood clearly. So it's only a dependency issue, as you mentioned. I changed the recipe to `noarch: python`.